src/utils/ETools/audio.cpp

```diff
@@ -250,6 +252,7 @@ int aiff_open(FILE* in, oe_enc_opt* opt, unsigned char* buf, int buflen)
     if (fread(buffer, 1, len, in) < len)
     {
         fprintf(stderr, _("Warning: Unexpected EOF in reading AIFF header\n"));
+        free(aiff);
```

Comment:
We can just move the `aiff` allocation down to the place where it's actually used.

`aiff->bigendian =` can be replaced with `int bigendian =`

Reply:
Unfortuantely, `aiff` is used in nested block, too. 

May be use `make_unique` + `release` where needed?

Reply:
Nevermind, fixed by `int bigendian =`